### PR TITLE
Branch ashley addcommand

### LINKED ISSUE #3: As a tech savvy warehouse manager, I want to be able to add my stock to the application so that I can record new stocks.

### LINKED ISSUE #14: As a multi-device user, I want to transport data from one device to another, so that I will not have to key in items one by one again.

### LINKED ISSUE #18: As a collaborative user, I want my inventory to be able to be shared with my collaborators, so that my collaborators can only read and find data.

### LINKED ISSUE #37: Implement the Add stock command
Augment the ab3 codebase to support the adding of new stock. Includes changing the names for the respective classes.

### PR DESCRIPTION
**Pull request by Ashley-Lau**
Reorganized code to suit warenager requirements. Hard coded serial number to a fixed number for now as the storage for a universal counter might take a little time. Also did not remove some commands as they might be relevant to the commands you are implementing. I also didnt touch the test files yet so they are probably not going to work. Don't be too harsh on me if there are any mistakes!

### CODE REVIEW

File: docs/DevOps.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ Any warnings or errors will be printed out to the console.
 
 Here are the steps to create a new release.
 
-1. Update the version number in [`MainApp.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/MainApp.java).
+1. Update the version number in [`MainApp.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/location/MainApp.java).
```
Comment (AudreyFelicio) [inline docs/DevOps.md]: Is all the `location` a placeholder for now?

File: LICENSE [hunk 2]
```diff
@@ -2,11 +2,11 @@ MIT License
 
 Copyright (c) 2016 Software Engineering Education - FOSS Resources
 
-Permission is hereby granted, free of charge, to any person obtaining a copy
+Permission is hereby granted, free of charge, to any stock obtaining a copy
```
Comment (AudreyFelicio) [inline LICENSE:5]: Why is all `person` changed to `stock`?

File: docs/DevOps.md (hunk 1 above)
Comment (Ashley-Lau) [reply docs/DevOps.md]: Nope. I think the refactoring screwed up all the links within the docs. Good catch!

File: LICENSE (hunk 2 above)
Comment (Ashley-Lau) [reply LICENSE:5]: We would be storing stocks rather than person in our warenager so I thought the renaming of the class would be appropriate here.

**Review by AudreyFelicio — APPROVED** [review]
Looks good to merge!

**Pull request merged**